tools/topology/topology1/sof/pipe-demux-eq-iir-playback.m4

```diff
@@ -19,9 +19,9 @@ dnl I.e. row index is the input channel, 1 means it is copied to
 dnl corresponding output channel (column index), 0 means it is discarded.
 dnl There's a separate matrix for all outputs.
 define(band_split_matrix1, `ROUTE_MATRIX(1,
-			     `BITS_TO_BYTE(1, 0, 0 ,0 ,0 ,0 ,0 ,0)',
 			     `BITS_TO_BYTE(1, 0, 0 ,0 ,0 ,0 ,0 ,0)',
 			     `BITS_TO_BYTE(0, 1, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(1, 0, 0 ,0 ,0 ,0 ,0 ,0)',
```

Comment:
this looks like a product-specific change in a generic macro definition, is this really desired?

@singalsu @juimonen can you comment on this?

Reply:
we used this order for alc1011 on CML platform before and max98390 on recent project.

Reply:
If desired this could be defined at one level higher up topology. I recall I placed the matrix into the pipeline macro just for simplicity.

Reply:
right, so something with ifndef(band_split_matrix, define band_split_matrix) to allow for a product-specific override of a default?